ophyd/device.py

```diff
@@ -812,7 +812,7 @@ def _initialize_device(cls):
 
         # The namedtuple associated with the device
         cls._device_tuple = namedtuple(f'{cls.__name__}Tuple',
-                                       [comp for comp in cls.component_names
+                                       [comp for comp in cls.component_names[:254]
```

Comment:
I have no guess about why this would be sliced. Can you add a comment?

Reply:
Will do. `namedtuple` is limited to 254 items, believe it or not. That means that prior to this commit, any device with more than that could not even be created in ophyd.

Reply:
This is 'fixed' in py3.7+ (https://stackoverflow.com/a/48051450/380231)